test/services/dumpers/agent-nodejs/agent-nodejs-dumper-mongoose-models.test.ts

```diff
@@ -47,114 +48,119 @@ describe('services > dumpers > agentNodejsDumper > mongoose models', () => {
         dbConfig: config.dbConfig,
         forestAuthSecret: 'forestAuthSecret',
         forestEnvSecret: 'forestEnvSecret',
+        language: Languages.Javascript,
       },
       schema,
     );
   }
 
-  describe('dump model files', () => {
-    describe('dumping schema', () => {
-      const testCases = [
-        {
-          name: 'should handle deep nested fields',
-          schema: deepNested,
-          file: {
-            model: 'persons',
-            expectedFilePath: `${__dirname}/expected/mongo-models/deep-nested.expected.js`,
+  const languages = [{ name: Languages.Javascript, extension: 'js' }];
```

Comment:
If you follow my previous comment regarding the Language enum, you'll see that it fits perfectly there